airflow/dags/oncokb_curation.py

```diff
@@ -0,0 +1,136 @@
+from datetime import timedelta, datetime
+from airflow.decorators import dag, task
+from airflow.io.path import ObjectStoragePath
+import pandas as pd
+import logging
+
+#Read data locally, we can change it to msk API later
+local = "path/to/folder/with/msk/data"
+base = ObjectStoragePath("s3://my_aws_s3@oncokb-ge")
+filter_path = "path/to/data"
+
+default_args = {
+    	'owner': 'oncoKB',
+		'retries': 3,
+		'retry_delay': timedelta(minutes=5)
+}
+
+@dag(
+    default_args=default_args,
+    dag_id="oncokb_curation_v1",
+    description="Get the statistic file and save to S3",
+    start_date=datetime(2024,6,8,0),
+    schedule_interval="@weekly",
+    catchup=False,
+)
+
+def curate_mutation_data():
+
+    def exclude_samples(data,filter_path):
+        excluded_sample_ids = pd.read_csv(filter_path,delimiter='\t',header=None)[0].tolist()
+        filtered_data = data[~data['SAMPLE_ID'].isin(excluded_sample_ids)]
+        return filtered_data
+    
+    def upload_to_s3(path,df):
+        with path.open("w") as file:
+            df.to_csv(file, sep='\t', index=False)
+
+    def read_from_s3(path: ObjectStoragePath):
+        with path.open() as f:
+            df = pd.read_csv(f,sep='\t')
+        return df
+
+    @task
+    def get_statistic_file():
+        test_name="msk_impact_2017"
+        file_path = local+test_name
+
+        # get the mutation data
+        sample = pd.read_csv(file_path+"/data_clinical_sample.txt", usecols=['SAMPLE_ID', 'CANCER_TYPE','CANCER_TYPE_DETAILED'], delimiter='\t', skiprows=4)
+        mutation = pd.read_csv(file_path+"/data_mutations.txt", usecols=['Hugo_Symbol', 'Tumor_Sample_Barcode','HGVSp_Short'], delimiter='\t',skiprows=1)
```

Comment:
Please use [`os.path.join`](https://www.geeksforgeeks.org/python-os-path-join-method/) to build out file paths. It makes your code less like to break.